data/org.freedesktop.portal.FileChooser.xml

```diff
@@ -28,8 +28,7 @@
       backend will present the user with a file chooser dialog.
 
       The selected files will be made accessible to the application
-      via the document portal, and the returned URI will point
-      into the document portal fuse filesystem in ``/run/user/$UID/doc/``.
+      (which may involve adding it to the document portal).
```

Comment:
Maybe add a link to the Document portal here?

Reply:
How would I do that?  If I write ``:ref:`Document portal<org.freedesktop.portal.Document>` `` in this section, it just generates `<span class="xref std std-ref">Document portal</span>`, with no hyperlink.

Reply:
Ah, got it — it's the Document**s** portal, in the plural.

Reply:
Done.